generator/ProtocolTestsGenerator/smithy-dotnet-protocol-test/smithy-build.json

```diff
@@ -0,0 +1,131 @@
+{
+  "version": "1.0",
+  "projections": {
```

Comment:
there are also other service specific tests, e.g., https://github.com/smithy-lang/smithy/tree/main/smithy-aws-protocol-tests/model/restXml/services, https://github.com/smithy-lang/smithy/tree/main/smithy-aws-protocol-tests/model/restJson1/services that should be included.